packages/website/package.json

```diff
@@ -1,14 +1,15 @@
 {
     "name": "@brisk-docs/website",
     "version": "1.0.0",
-    "description": "The Brisk website documenter", 
+    "description": "The Brisk website documenter",
     "main": "index.js",
     "license": "MIT",
     "private": true,
     "scripts": {
         "test": "jest",
         "dev": "node bin/index.js start --config=dummy-data/simple-project/docs.config.js",
-        "dev:webpack": "node bin/bin.js start --config=dummy-data/custom-webpack-project/docs.config.js",
+        "dev:webpack": "node bin/index.js start --config=dummy-data/custom-webpack-project/docs.config.js",
+        "dev:typescript": "node bin/index.js start --config=dummy-data/typescript-project/docs.config.js",
```

Comment:
What's with the choice for these to be in dummy-data not fixtures?

Followup point: If we write fixtures, should we also be okay running against them? Is there a clear ontological difference between them?

Reply:
Tbh they should probably be written as tests and not even be put in dummy_data at all